addon/templates/components/frost-link.hbs

```diff
@@ -1,26 +1,26 @@
 {{#if (eq design 'info-bar')}}
-  <div class='content '>
-    <div class=' text'>
-      <div class='svg'>{{frost-icon icon=icon}}</div>
-      {{yield}}
-      </div>
-  </div>
+    <div class='content '>
+        <div class=' text'>
+            <div class='svg'>{{frost-icon icon=icon}}</div>
+          {{#if linkTitle}}{{linkTitle}}{{else}}{{yield}}{{/if}}
```

Comment:
Where did `linkTitle` come from? If this is a new optional property please add it to the links propTypes as well as documentation in the dummy app.